### ISSUE TITLE
⌨ `EmployeeProfilePage.UnverifiedUser` story insufficient color contrast

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#17103]
### ⌨ Accessibility and Expected Behaviour

Join a community text has insufficient color contrast in light and dark mode.

https://61e099a1bb1465003a73d3ce-ubitoimoar.chromatic.com/?path=/story/pages-employeeprofile-employeeprofilepage--unverified-user

### WCAG 2.1 Criteria

https://dequeuniversity.com/rules/axe/4.12/color-contrast?application=axeAPI

### 📋 Steps to Reproduce

1. `pnpm storybook`
2. https://61e099a1bb1465003a73d3ce-ubitoimoar.chromatic.com/?path=/story/pages-employeeprofile-employeeprofilepage--unverified-user
3. Accessibility tab has errors 

### 🕵️ Details

https://github.com/GCTC-NTGC/gc-digital-talent/blob/85a6975371bc9b6216fcbe812dbd73c27413f1f5/apps/web/src/components/EmployeeVerificationSection/EmployeeVerificationSection.tsx#L385-L391

### 📸 Screenshot

<img width="1715" height="825" alt="Image" src="https://github.com/user-attachments/assets/106943b0-a90d-4f11-88c4-717e3a67cac8" />

### ✅ Acceptance Criteria

- [ ] `EmployeeProfilePage.UnverifiedUser` story does not have a11y errors
